CRAMv3.tex

```diff
@@ -600,55 +605,30 @@ \subsubsection*{Preservation map}
 \subsubsection*{Data series encodings}
 
 Each data series has an encoding. These encoding are stored in a map with byte[2] 
-keys:
+keys and are decoded in approximately this order\footnote{The precise order is defined in section~\ref{sec:record}.}:
 
 \begin{tabular}{|l|l|>{\raggedright}p{100pt}|>{\raggedright}p{220pt}|}
 \hline
 \textbf{Key} & \textbf{Value data type} & \textbf{Name} & \textbf{Value}\tabularnewline
 \hline
-BF & encoding\texttt{<}int\texttt{>} & bit flags & see separate section\tabularnewline
+BF & encoding\texttt{<}int\texttt{>} & BAM bit flags & see separate section\tabularnewline
 \hline
-AP & encoding\texttt{<}int\texttt{>} & in-seq positions & 0-based alignment start 
-delta from previous record *\tabularnewline
+CF & encoding\texttt{<}int\texttt{>} & CRAM bit flags & see specific section\tabularnewline
 \hline
-FP & encoding\texttt{<}int\texttt{>} & in-read positions & positions of the read 
-features\tabularnewline
+RI & encoding\texttt{<}int\texttt{>} & reference id & record reference id from
+the BAM file header\tabularnewline
 \hline
 RL & encoding\texttt{<}int\texttt{>} & read lengths & read lengths\tabularnewline
 \hline
-DL & encoding\texttt{<}int\texttt{>} & deletion lengths & base-pair deletion lengths\tabularnewline
-\hline
-NF & encoding\texttt{<}int\texttt{>} & distance to next fragment & number of records 
-to the next fragment*\tabularnewline
-\hline
-BA & encoding\texttt{<}byte\texttt{>} & bases & bases\tabularnewline
-\hline
-QS & encoding\texttt{<}byte\texttt{>} & quality scores & quality scores\tabularnewline
-\hline
-BB & encoding\texttt{<}byte[ ]\texttt{>} & stretches of bases & bases\tabularnewline
-\hline
-QQ & encoding\texttt{<}byte[ ]\texttt{>} & stretches of quality scores & quality scores\tabularnewline
-\hline
-FC & encoding\texttt{<}byte\texttt{>} & read features codes & see separate section\tabularnewline
-\hline
-FN & encoding\texttt{<}int\texttt{>} & number of read features & number of read 
-features in each record\tabularnewline
-\hline
-BS & encoding\texttt{<}byte\texttt{>} & base substitution codes & base substitution 
-codes\tabularnewline
-\hline
-IN & encoding\texttt{<}byte[ ]\texttt{>} & insertion & inserted bases\tabularnewline
+AP & encoding\texttt{<}int\texttt{>} & in-seq positions & 0-based alignment start
+delta from previous record *\tabularnewline
 \hline
 RG & encoding\texttt{<}int\texttt{>} & read groups & read groups. Special value 
 `-1' stands for no group.\tabularnewline
 \hline
-MQ & encoding\texttt{<}int\texttt{>} & mapping qualities & mapping quality scores 
- \tabularnewline
-\hline
-TL & encoding\texttt{<}int\texttt{>} & tag ids  & list of tag ids, see tag encoding 
-section\tabularnewline
+RN\footnote{Note this is decoded after MF if ...} & encoding\texttt{<}byte[ ]\texttt{>} & read names & read names\tabularnewline
```

Comment:
I don't see this footnote in the rendered PDF.  Same problem for TL.

Reply:
Thanks.  I've reformatted these using LaTeX threeparttable.